tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_sort_contacts.py

```diff
@@ -24,20 +22,9 @@ def setUp(self):
         self.sorted_contacts_name_by_last = sorted(self._contacts_name_list, key=lambda name: name[1])
 
     def test_sort_contacts(self):
-        order_by_last_name_default = False
-
         contacts_app = Contacts(self.marionette)
         contacts_app.launch()
 
-        # if "Order by last name" switch is on, turn off it
-        contacts_settings = contacts_app.tap_settings()
-        self.assertEqual(contacts_settings.order_by_last_name, order_by_last_name_default)
-        # This test starts from when order_by_last_name is false
-        if order_by_last_name_default:
-            contacts_settings.tap_order_by_last_name()
-        contacts_settings.tap_done()
-
-        # here we wait for contacts once the correct order has been set
```

Comment:
This test had the flexibility that you could just flip the order_by_last_name_default boolean to have the test pass again if order by last name would be suddenly changed.
But I don't think it's a big deal, this is some nice clean-up and when the order by last name thing suddenly changes, then it still should be quite easy to fix the test.